rest-service/manager_rest/resource_manager.py

```diff
@@ -964,6 +971,10 @@ def _execute_system_workflow(self,
             if deployment:
                 execution.set_deployment(deployment)
 
+        if no_op:
```

Comment:
Can it be done only in the plugins update logic? because this is a general place and no_op is not available mode for other wf

Reply:
Well, I'd have to copy code from this function into the "update_plugin" function, or at least break this function to other functions which is more of a mess IMO

Reply:
I think you just need to use the data model and write it, no?

Reply:
I could, just wanted the execution to have as many 'real' parameters as it can have so I won't break inconsistencies in the DB.